web/src/App.tsx

```diff
@@ -1,141 +1,10 @@
-  width: 100%;
-  border: 1px solid #f2f2f2;
-  padding: 10px;
-  margin-bottom: 10px;
-`
-
-const StyledButton = styled.button`
-  background: palevioletred;
-  color: #fff;
-  border-radius: 3px;
-  border: 2px solid palevioletred;
-  padding: 3px 10px;
-  outline: none;
-  cursor: pointer;
-`
-const ItemWrapper = styled.div`
-  display: flex;
-  padding: 4px;
-`
-
-const ItemTitle = styled.div`
-  flex: 1 1 auto;
-`
-
-const App = () => {
-  const { tokenData, token } = useContext(AuthContext)
-  const [todos, setTodos] = useState<GetTodoAllResponse[]>([])
-  const [value, setValue] = React.useState('')
-
-  const todoAPI = new TodoAPI(token)
-
-  useEffect(() => {
-    todoAPI
-      .getAll()
-      .then((response) => setTodos(response.data))
-      .catch((e) => {})
-  }, [])
-
-  const addItem = (event: any) => {
-    event.preventDefault()
-    if (!value) return
-
-    todoAPI
-      .create({
-        addTodoRequest: {
-          title: value,
-        },
-      })
-      .then((response) => {
-        const item: AddTodoResponse = response.data
-        const items = [...todos, item]
-        // @ts-ignore
-        setTodos(items)
-      })
-  }
-
-  const handleRemove = (id: string, index: number) =>
-    todoAPI.deleteById({ id: id }).then((response) => {
-      const newTodos = [...todos]
-      newTodos.splice(index, 1)
-      setTodos(newTodos)
-    })
-
-  const handleToggle = (id: string, is_completed: boolean, index: number) => {
-    todoAPI
-      .updateById({
-        id: id,
-        updateTodoRequest: {
-          is_completed: !is_completed,
-          title: todos[index].title,
-        },
-      })
-      .then((response) => {
-        const items = todos
-        let item = items[index]
-        item.is_completed = !is_completed
-        items[index] = item
-        setTodos([...items])
-      })
-  }
-
+function App() {
   return (
-    <ToDoContainer>
-      <Header>Awesome Boilerplate</Header>
-      <div>Logged inn as: {token && tokenData?.['unique_name']}</div>
-      <div className="form">
-        <form onSubmit={addItem}>
-          <Input
-            className="input"
-            placeholder="Add Task"
-            onChange={(e) => setValue(e.target.value)}
-          />
-          <StyledButton className="add-button" type="submit">
-            Add
-          </StyledButton>
-        </form>
-      </div>
-      {todos.map((todo, index) => (
-        <ToDoItem
-          is_completed={todo.is_completed}
-          key={`${index}-${todo.is_completed}`}
-          onClick={() => handleToggle(todo.id, todo.is_completed, index)}
-        >
-          <ItemWrapper>
-            <ItemTitle>{todo.title}</ItemTitle>
-            <StyledButton onClick={() => handleRemove(todo.id, index)}>
-              Remove
-            </StyledButton>
-          </ItemWrapper>
-        </ToDoItem>
-      ))}
-    </ToDoContainer>
+    <div className="App">
+      <p>
+        Edit <code>src/App.tsx</code> and save to reload.
```

Comment:
Could be done in another PR.
But adding a simple router with a `/thisisasublocation` for reference would be nice to have in the template

Reply:
Agree. I think we should do a separate PR for that.